sklearn/linear_model/sgd_fast_helpers.h

```diff
@@ -3,7 +3,11 @@
 #ifdef _MSC_VER
 # include <float.h>
 # define skl_isfinite _finite
+# define skl_isfinite32 _finite
```

Comment:
Could you add a comment saying why this change is needed?

```
# When re-declaring the functions in the template for cython
# specific for each parameter input type, it needs to be 2 different functions
# as cython doesn't support function overloading.
```

@jnothman can you help in deciding whether this comment is needed or this is obvious.